gcovr/formats/html/default/functions_page.content.html

```diff
@@ -1,11 +1,12 @@
 {# -*- engine: jinja -*- #}
+{% for entry in all_functions %}
+{% if loop.index0 % 10000 == 0 %}
```

Comment:
Instead of checking the index and emitting `<table>`/`</table>` tags as necessary, this is probably much clearer to express with the `batch` filter: https://jinja.palletsprojects.com/en/3.1.x/templates/#jinja-filters.batch

Roughly:

```jinja
{% for function_batch in all_functions | batch(10000) %}
<table ...>
{% for entry in function_batch %}
...
{% endfor %}
</table>
{% endfor %}
```

Reply:
I didn't know this filter. It's really clearer.